### ISSUE TITLE
SyntaxError: Unexpected token ' in JSON at position 0

### ISSUE THREAD

**Issue by minooo** [zeit/next.js#3994]
https://github.com/zeit/next.js/blob/b02e5849e9472c80839704b0003448f30482a818/examples/custom-server-typescript/nodemon.json#L4

SyntaxError: Unexpected token ' in JSON at position 0

**Comment by iamglenbacs** [comment]
did you resolve this?

**Comment by minooo** [comment]
not yet，I just give up use TS - - !

**Comment by timneutkens** [comment]
@kachkaev could you check this 🙏

**Comment by kachkaev** [comment]
@minooo what command did you run when the error occurred? Could you please share a bit more of your output?

**Comment by Dominic-Preap** [comment]
@kachkaev when I type `yarn dev` from [examples/custom-server-typescript](https://github.com/zeit/next.js/tree/canary/examples/custom-server-typescript), I get this error.

![image](https://user-images.githubusercontent.com/14802170/37519081-6d6ad43c-294a-11e8-97a0-96d9203eef7c.png)

**Comment by kachkaev** [comment]
Hmm, could be something to do with quotes on Windows 🤔 I'm not sure how exactly to fix this as I don't have a machine on which I could reproduce this.

Can you try playing with nested quotes in `package.json`? E.g. smth like:

```
"ts": "ts-node --compilerOptions \"{\\\"module\\\":\\\"commonjs\\\"}\"" 
```
etc.

**Comment by Dominic-Preap** [comment]
@kachkaev Sorry for late response. It works after replacing the command. Thank you. Suck to be Windows user 😂😂😂

**Comment by minooo** [comment]
Thank you, evereone!

**Comment by shreya121212** [comment]
SyntaxError: Unexpected token p in JSON at position 2
    at JSON.parse (<anonymous>)
    at Object.Utils.JSON.parse (/Users/Admin/Desktop/bitshare/node_modules/jayson/lib/utils.js:274:26)
    at IncomingMessage.<anonymous> (/Users/Admin/Desktop/bitshare/node_modules/jayson/lib/client/http.js:75:22)
    at IncomingMessage.emit (events.js:187:15)
    at endReadableNT (_stream_readable.js:1081:12)
    at process._tickCallback (internal/process/next_tick.js:63:19)

this is the errror which comes from bitshare-rpc, how can i resolve this??

**Comment by kqadem** [comment]
> did anyone resolve this?

The solution from @kachkaev worked for me.

**Comment by Glenzy** [comment]
So escaping the quotes works. However does this mean it will break in team environments where everyone else is using OSX machines?

**Comment by PPjaisri** [comment]
SyntaxError: Unexpected token / in JSON at position 0
    at JSON.parse (<anonymous>)
    at Object.Module._extensions..json (internal/modules/cjs/loader.js:795:27)
    at Module.load (internal/modules/cjs/loader.js:653:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:593:12)
    at Function.Module._load (internal/modules/cjs/loader.js:585:3)
    at Module.require (internal/modules/cjs/loader.js:690:17)
    at require (internal/modules/cjs/helpers.js:25:18)
    at Object.<anonymous> (C:\Users\best\Desktop\express_exceed\server.js:14:15)
    at Module._compile (internal/modules/cjs/loader.js:776:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:787:10)
[nodemon] app crashed - waiting for file changes before starting...

How can I solve it ??

**Comment by allenbert** [comment]
Hi @kachkaev I didn't use any command line in package json. Locally my program runs when it deployed i getting the Error like:
SyntaxError: Unexpected token - in JSON at position 0
    at JSON.parse (<anonymous>)
    at createStrictSyntaxError (/srv/api/node_modules/body-parser/lib/types/json.js:158:10)
    at parse (/srv/api/node_modules/body-parser/lib/types/json.js:83:15)
    at /srv/api/node_modules/body-parser/lib/read.js:121:18
    at invokeCallback (/srv/api/node_modules/raw-body/index.js:224:16)
    at done (/srv/api/node_modules/raw-body/index.js:213:7)
    at IncomingMessage.onEnd (/srv/api/node_modules/raw-body/index.js:273:7)
    at emitNone (events.js:111:20)
    at IncomingMessage.emit (events.js:208:7)
    at endReadableNT (_stream_readable.js:1064:12)
    at _combinedTickCallback (internal/process/next_tick.js:139:11)
    at process._tickCallback (internal/process/next_tick.js:181:9)

**Comment by torsami77** [comment]
if using fetch, check your headers content-Type, you might want to remove it

**Comment by balazsorban44** [comment]
This issue has been automatically locked due to no recent activity. If you are running into a similar issue, please create a new issue with the steps to reproduce. Thank you.
